Dockerfile.must-gather

```diff
@@ -1,17 +1,9 @@
-FROM quay.io/openshift/origin-must-gather:4.18.0 AS builder
+FROM quay.io/openshift/origin-cli:latest
```

Comment:
we should pick the latest _of the corrisponding Y stream_ not the latest in general